guides/source/configuring.md

```diff
@@ -1615,15 +2117,25 @@ Initialization events
 
 Rails has 5 initialization events which can be hooked into (listed in the order that they are run):
 
-* `before_configuration`: This is run as soon as the application constant inherits from `Rails::Application`. The `config` calls are evaluated before this happens.
+#### before_configuration
+
+This is run as soon as the application constant inherits from `Rails::Application`. The `config` calls are evaluated before this happens.
+
+#### before_initialize
 
-* `before_initialize`: This is run directly before the initialization process of the application occurs with the `:bootstrap_hook` initializer near the beginning of the Rails initialization process.
+This is run directly before the initialization process of the application occurs with the `:bootstrap_hook` initializer near the beginning of the Rails initialization process.
 
-* `to_prepare`: Run after the initializers are run for all Railties (including the application itself), but before eager loading and the middleware stack is built. More importantly, will run upon every code reload in `development`, but only once (during boot-up) in `production` and `test`.
+#### to_prepare
 
-* `before_eager_load`: This is run directly before eager loading occurs, which is the default behavior for the `production` environment and not for the `development` environment.
+Run after the initializers are run for all Railties (including the application itself), but before eager loading and the middleware stack is built. More importantly, will run upon every code reload in `development`, but only once (during boot-up) in `production` and `test`.
 
-* `after_initialize`: Run directly after the initialization of the application, after the application initializers in `config/initializers` are run.
+#### before_eager_load
+
+This is run directly before eager loading occurs, which is the default behavior for the `production` environment and not for the `development` environment.
+
+#### after_initialize
+
+Run directly after the initialization of the application, after the application initializers in `config/initializers` are run.
 
 To define an event for these hooks, use the block syntax within a `Rails::Application`, `Rails::Railtie` or `Rails::Engine` subclass:
```

Comment:
This paragraph now appears to fall under the "after_initialize" subheading (whereas it was previously distinct from the "after_initialize" list item).  I'm not sure how to address that. :thinking:  Perhaps this section's list items should remain as list items for now.

Reply:
👍

Reply:
I've left this out for now.